stories/components/Dialog/Dialog.md

```diff
@@ -0,0 +1,72 @@
+# Dialog
+
+**Available components**:
+
+- [DialogCentreScreen](#DialogCentreScreen)
+- [DialogFullScreen](#DialogFullScreen)
+
+**How it is built**
+
+- Dialogs are a mainly just a wrapper which is passed a styled child component.
+
+## DialogCentreScreen
+
+Accessible `Dialog` component.
+A centred white modal box surrounded by a light-grey coloured background.
+
+### Props
+
+| name     | type      | default | description             |
+| -------- | --------- | ------- | ----------------------- |
+| label    | string    |         | Aria-label              |
+| children | Component |         | Placeholder for input   |
+| dialog   | useDialog |         | Hook from reakit/Dialog |
+
+### Example
+
+```javascript
+import { useDialog } from "reakit/Dialog";
```

Comment:
Shouldn't it be importing from `@yuhuinc/yuhui`? 😅

Reply:
Thanks for the catch!